packages/bot-web-ui/src/components/summary/summary-card.tsx

```diff
@@ -25,12 +25,13 @@ const SummaryCard = ({
     server_time,
     setCurrentFocus,
 }: TSummaryCardProps) => {
+    const is_cryptocurrency = contract_info?.underlying.startsWith('cry') || false;
```

Comment:
there is a method in deriv/shared to check whether the currency is_crypto `isCryptocurrency` can we use this ?

Reply:
I've changed it with isCryptocurrency() from shared package:
```
    const currency = contract_info?.display_name.split('/')[0];
    const is_cryptocurrency = contract_info?.display_name ? isCryptocurrency(currency as string) : false;
```

But  I have my doubts about the use of this method in this particular case. It may be more costly and time-consuming. We need to determine if this market is a cryptocurrency or not. In the previous variant we took one variable out of the store and determined. But using method from shared package we first take variable from store and then pass it to function, function has another function that goes to object and checks if this currency is in cryptocurrency list or not... Perhaps this is a more costly and slower to calculate operation.

@prince-deriv @rupato-deriv what do you think about it?

Reply:
P.S. we cant use in current case currency from client.currency (client store) as we use in others packages. It is account currency, we need market currency for trading.

Reply:
@maryia-matskevich-deriv yes we can use the client store in our dbot-package  the stores are accessble through the connect function because of the passthrough.... 
you can use this check but if we use what is coming through shared because it checks for many conditions like the currency type too... 
also in this current check we are directly checking if the underlying is giving us crypto which is right but we need to make sure it gives crypto all times.